Runtime exceptions (missing dependency?) on fresh build, OS X

Hey all, I'm trying to build inky on OS X (sierra, 10.12.6) from a fresh clone.
Node v6.11.2, npm v3.10.10

After cloning and running `./BUILD_FOR_MAC.command`, and launching inky...
I can open an ink file, but nothing runs in the right panel, and I can't quit or open another ink file in the left panel.

I get this exception in the inky console after I launch:
```
Uncaught Error: Cannot find module 'fuzzaldrin-plus'
    at Module._resolveFilename (module.js:470:15)
    at Function.Module._resolveFilename (/Volumes/Inky 1/Inky.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Volumes/Inky 1/Inky.app/Contents/Resources/app.asar/renderer/goto.js:5:31)
    at Object.<anonymous> (/Volumes/Inky 1/Inky.app/Contents/Resources/app.asar/renderer/goto.js:426:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
```
Looks like it's trying to find the `fuzzaldrin` module but it can't?

Strange because I see fuzzaldrin-plus in app/node_modules/.
`RUN.command` gives the same runtime error.

--- a/app/package.json
+++ b/app/package.json
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "chokidar": "^1.6.0",
-    "fuzzaldrin-plus": "https://github.com/joethephish/fuzz-aldrin-plus/archive/patch1.tar.gz",
+    "fuzzaldrin-plus": "^0.4.1",
     "inkjs": "^1.6.0",
     "lodash": "^4.13.1",
     "mkdirp": "^0.5.1",

--- a/app/package.json
+++ b/app/package.json
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "chokidar": "^1.6.0",
-    "fuzzaldrin-plus": "^0.4.1",
+    "fuzzaldrin-plus": "git+https://github.com/awmcclain/fuzz-aldrin-plus.git",
     "inkjs": "^1.6.0",
     "lodash": "^4.13.1",
     "mkdirp": "^0.5.1",

